lib/datasource/pypi/index.ts

```diff
@@ -1,7 +1,7 @@
 import url from 'url';
 import changelogFilenameRegex from 'changelog-filename-regex';
 import { logger } from '../../logger';
-import { parse } from '../../util/html';
+import { HTMLElement, parse } from '../../util/html';
```

Comment:
(a) how was this used previously if it wasn't imported?
(b) is it needed explicitly here anyway, if the function declares its return type?

Reply:
It's because the global `HTMLElement` is incompatible to the on of `node-html-parser`